### ISSUE TITLE
Getting Accessibility error on feedback form.

### ISSUE THREAD

**Issue by Shalini-git25** [DFE-Digital/find-information-products-services-frontend#174]
**Describe the bug**
On feedback form, ran tests using 'ARC toolkit' to test accessibility issues. Showing accessibility error as 'ARIA attribute is not allowed'.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to any page on FIPS application.
2. Click on "Give feedback or report a problem with this page" link at the bottom of the page.
3. Feedback form opens where user can provide their feedback.
4. Run tests using ARC Toolkit - error message is displayed as 'ARIA attribute is not allowed'.

**Expected behaviour**
No accessibility error should be present on the feedback form area.

**Screenshots**

<img width="2235" height="1367" alt="Image" src="https://github.com/user-attachments/assets/033e2571-6fc5-4ce9-b143-a89f6562c29a" />

**Additional context**
